projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -1041,6 +1041,8 @@ For examples of how to define and use new extension points, please see the code
 ++ speech ++[speechExtPts]
 || Type | Extension Point | Description |
 | ``Action`` | ``speechCanceled`` | Triggered when speech is canceled. |
+| ``Action`` | ``pre_speechCanceled`` | Triggered before speech is canceled. |
+| ``Action`` | ``pre_speech`` | Triggered when NVDA is about to speak something. |
```

Comment:
Since this action fires even when speech mode is off, I think this is more accurate

```suggestion
| ``Action`` | ``pre_speech`` | Triggered before NVDA handles prepared speech. |
```